Level Solutions/20176.0.txt

```diff
@@ -0,0 +1,37 @@
+@0
+
+#If number is positive the "NEG" command makes it negative and vice versa.

```

Comment:
Again, explain "why" -- why is changing a positive to a negative a useful thing to do in this program?